_posts/en/newsletters/2021-05-26-newsletter.md

```diff
@@ -0,0 +1,266 @@
+Japanese, and [Akio Nakamura][] who has also translated and reviewed our
+Japanese material.
+
+Thanks to all the members of the Bitcoin community -- too numerous to name
+individually -- who have reviewed our newsletters, helped us understand
+concepts, and opened issues and PRs when we've made mistakes.
+
+All of this work is made possible by our generous [supporters][], primarily
+our [founding sponsors][] - Wences Casares, John Pfeffer and Alex Morcos.
+
+Finally, thank you, our readers. We love being part of this community and
+contributing to this ecosystem. Knowing how valuable this resource is to so many
+people, and hearing feedback from our readers is hugely rewarding for us. If you
+want to contribute, or have suggestions for how we can improve, please don't
+hesitate to contact us at [info@bitcoinops.org][info].
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Why are there more than two transaction outputs in a coinbase transaction?]({{bse}}105831)
+  Andrew Chow explains some common outputs in a coinbase transaction:
+
+    * a single miner block reward payment
+
+    * multiple payments, as with a mining pool paying miners
+
+    * [BIP141][bip141 commitment]'s `OP_RETURN` witness commitment
+
+    * additional `OP_RETURN` commitments, as in [merge mining][se 273 merge
+      mining] and other protocols
+
+- [fundrawtransaction - what is it?]({{bse}}105811)
+  Pieter Wuille illustrates what the `fundrawtransaction` RPC does by providing
+  4 examples of how to send coins using the RPC.
+
+- [What previously existing technologies made Bitcoin possible?]({{bse}}106000)
+  Murch provides a summary, based on the [Bitcoin's Academic Pedigree
+  paper][bitcoins academic pedigree paper], of the existing technological ingredients
+  that were combined to create Bitcoin. These technologies are linked
+  timestamping/verifiable logs, byzantine fault tolerance, proof of work,
+  digital cash, and public keys as identities.
+
+- [How can I follow the progress of miner signaling for Taproot activation?]({{bse}}105853)
+  In addition to Hampus Sjöberg's [https://taproot.watch][taproot watch website] website, Bitcoin
+  Core users can use `getblockchaininfo` to get a count of signaling blocks and
+  `getblock`'s versionhex field, where the signaling version bits reside, to
+  observe signaling.
+
+- [What block space savings would we get for coinjoins (and payjoins) if we had cross input signature aggregation?]({{bse}}106224)
```

Comment:
I was planning on writing a better answer for that one anyway, I can make sure that it happens today.

I would agree, though, that we might want to drop it otherwise.

Reply:
I look forward to reading it :)

We can either include it here or in next month's SE section. It's not going to go out of date.